external-crates/move/crates/move-compiler/src/verification/mod.rs

```diff
@@ -1,5 +0,0 @@
-// Copyright (c) The Diem Core Contributors
-// Copyright (c) The Move Contributors
-// SPDX-License-Identifier: Apache-2.0
-
-pub mod ast_filter;
```

Comment:
Can `ast_filter` go, too?

Reply:
We either have to error on the old `verify_only` attributes, or keep filtering sadly. 
I opted to preserving the filtering since it seemed the kinder thing